ddl/ingest/checkpoint.go

```diff
@@ -174,18 +178,23 @@ func (s *CheckpointManager) UpdateCurrent(taskID int, added int) error {
 	if !flushed || err != nil {
 		return err
 	}
+
+	s.mu.Lock()
+	defer s.mu.Unlock()
 	s.progressLocalSyncMinKey()
 	if imported && s.minKeySyncGlobal.Cmp(s.minKeySyncLocal) != 0 {
 		s.minKeySyncGlobal = s.minKeySyncLocal
 		s.globalCnt = s.localCnt
 		s.dirty = true
+
+		s.pidGlobal = s.pidLocal
```

Comment:
Why here we need to update global info? It seems we do it after `WriteLocal`, so why not update local info?

Reply:
Do import to tikv in https://github.com/pingcap/tidb/pull/44024/files#diff-451edd897e959340ba4a6565b82969ae58c35a4b9715674769565f3958c39a2dR177.